apps/watch/src/components/LanguageSwitchDialog/LanguageSwitchDialog.tsx

```
@@ -0,0 +1,178 @@
+        const localName = new Intl.DisplayNames([languageCode], {
+          type: 'language'
+        }).of(languageCode)
+
+        return {
+          languageCode,
+          nativeName: nativeName === localName ? undefined : nativeName,
+          localName: localName ?? ''
+        }
+      })
+    setLanguages(formattedLanguages)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)